pkg/sdkserver/alpha/sdkserver.go

```diff
@@ -26,3 +29,28 @@ type SDKServer struct{}
 func NewSDKServer() *SDKServer {
 	return &SDKServer{}
 }
+
+// PlayerConnect should be called when a player connects
+func (s SDKServer) PlayerConnect(ctx context.Context, id *alpha.PlayerId) (*alpha.Empty, error) {
+	panic("implement me")
+}
+
+//PlayerDisconnect should be called when a player disconnects
```

Comment:
nit: Add a space between // and the start of the comment

Reply:
Also a super nit, but comments should be formatted as sentences ending with a period.

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences